corehq/apps/hqmedia/templates/hqmedia/references.html

```diff
@@ -44,9 +44,47 @@
           </p>
           {% if multimedia_state.has_form_errors or multimedia_state.has_missing_refs %}
             <p>
-              <a href="#" class="btn btn-default" data-bind="click: toggleMissingRefs, text: toggleRefsText"></a>
+              <select-toggle data-apply-bindings="false"
+                             params="
+                               value: onlyMissing,
+                               options: [{
+                                 id: 'false',
+                                 text: '{% trans_html_attr 'All references' %}',
+                               }, {
+                                 id: 'true',
+                                 text: '{% trans_html_attr 'Only missing references' %}',
+                               }],
+                             "></select-toggle>
             </p>
           {% endif %}
+          <p>
+              <select-toggle data-apply-bindings="false"
+                             params="
+                               value: mediaClass,
+                               options: [{
+                                 id: 'all',
```

Comment:
nit: usually for dicts in a list i've formatted it as follows, to follow how it's formatted in json.
```
options: [
  {
    foo: bar,
  },
  {
    biz: baz,
  },
]
```

Reply:
3f11ac08db8